app/services/record_sort.rb

```diff
@@ -0,0 +1,22 @@
+class RecordSort
+  attr_reader :column, :order
+
+  def initialize
+    @column = default_column
+    @order = default_order
+  end
+
+  def update(column:, order:)
+    @column = valid_sort_columns.include?(column) ? column : @column
```

Comment:
I think this would be more readable:

```ruby
@column = column if valid_sort_columns.include?(column)
@order = order if valid_sort_orders.include?(order)
```